statsmodels/distributions/tests/test_ecdf.py

```diff
@@ -42,3 +43,13 @@ def test_monotone_fn_inverter(self):
         f = monotone_fn_inverter(fn, x)
         npt.assert_array_equal(f.y, x[::-1])
         npt.assert_array_equal(f.x, y[::-1])
+
+    def test_ecdf_discrete(self):
+        x = [3, 3, 1, 4]
+        e = ECDFDiscrete(x)
+        npt.assert_array_equal(e.x, [-np.inf, 1, 3, 4])
+        npt.assert_array_equal(e.y, [0, 0.25, 0.75, 1])
+        e1 = ECDFDiscrete([3.5, 3.5, 1.5, 1, 4])
+        e2 = ECDFDiscrete([3.5, 1.5, 1, 4], freq_weights=[2, 1, 1, 1])
```

Comment:
maybe add this example to docstring, mentioning that these are the two equivalent ways of defining the ecdf